crescent/internal/handle_resp.py

```diff
@@ -16,6 +16,7 @@
     Snowflake,
 )
 from hikari.api import InteractionResponseBuilder
+from hikari.impl import AutocompleteChoiceBuilder
```

Comment:
This feels hacky but I could not find any way to use this implementation any other way. There's no methods like there should be, such as "add_choice" - they're all just "set_choices", which require a list of already instantiated AutocompleteChoiceBuilders.

Reply:
if theres nothing we can do about it then don't worry about it. Ill make an issue in hikari for this.